opennms-base-assembly/src/main/filtered/etc/syslog/NetgearProsafeSmartSwitch.syslog.xml

```diff
@@ -0,0 +1,218 @@
+            <uei>uei.opennms.org/vendor/netgear/syslog/prosafesmartswitch/Warning/EnergyEfficientEthernetStateTrue</uei>
+            <parameter-assignment matching-group="1" parameter-name="portName" />
+        </ueiMatch>
+        <ueiMatch>
+            <facility>local7</facility>
+            <severity>Warning</severity>
+            <process-match expression="^%NT_GREEN-W-EeeLldpMultiNeighbours$" />
+            <match type="regex" expression="^Multiple LLDP neighbours on port (.*) - EEE operational state is FALSE$" />
+            <uei>uei.opennms.org/vendor/netgear/syslog/prosafesmartswitch/Warning/EnergyEfficientEthernetStateFalse</uei>
+            <parameter-assignment matching-group="1" parameter-name="portName" />
+        </ueiMatch>
+        <ueiMatch>
+            <facility>local7</facility>
+            <severity>Warning</severity>
+            <process-match expression="^%STP-W-PORTSTATUS$" />
+            <match type="regex" expression="^(.*) of instance ([0-9.]+): STP status Forwarding$" />
+            <uei>uei.opennms.org/vendor/netgear/syslog/prosafesmartswitch/Warning/PortSTPStatusForwarding</uei>
+            <parameter-assignment matching-group="1" parameter-name="portName" />
+            <parameter-assignment matching-group="2" parameter-name="instanceNumber" />
+        </ueiMatch>
+        <ueiMatch>
+            <facility>local7</facility>
+            <severity>Warning</severity>
+            <process-match expression="^%BOOTP_DHCP_CL-W-FLNMEEMPTY$" />
+            <match type="substr" expression="No option 67 in the DHCP packet. Unable to start autoconfiguration." />
+            <uei>uei.opennms.org/vendor/netgear/syslog/prosafesmartswitch/Warning/UnableToStartDHCPAutoconfiguration</uei>
+        </ueiMatch>
+        <ueiMatch>
+            <facility>local7</facility>
+            <severity>Warning</severity>
+            <process-match expression="^%LINK-W-Down$" />
+            <match type="regex" expression="^(.*)$" />
+            <uei>uei.opennms.org/vendor/netgear/syslog/prosafesmartswitch/Warning/PortStatusLinkDown</uei>
+            <parameter-assignment matching-group="1" parameter-name="portName" />
+        </ueiMatch>
+        <ueiMatch>
+            <facility>local7</facility>
+            <severity>Warning</severity>
+            <process-match expression="^%AAA-W-REJECT$" />
+            <match type="regex" expression="^New http connection for user (.+), source ([0-9.]+) destination ([0-9.]+)  REJECTED$" /> <!-- two spaces needed -->
+            <uei>uei.opennms.org/vendor/netgear/syslog/prosafesmartswitch/Warning/WebConsoleLoginFailure</uei>
+            <parameter-assignment matching-group="1" parameter-name="userName" />
+            <parameter-assignment matching-group="2" parameter-name="sourceIP" />
+            <parameter-assignment matching-group="3" parameter-name="destinationIP" />
+        </ueiMatch>
+        <!-- catch-all event for other Warning syslog messages -->
```

Comment:
This definition is too generic. It is bound to match messages from non-Netgear devices. Suggest making it more specific or removing it altogether.